Gregam3/ASEGroup1
# Rp 236 tests

### Description
This PR is linked to issue #236 
Added many tests for the SPARQL Query generator

### Which areas of Code needs reviewing
- [ ] Swagger 

### Where should the reviewer start (Optional)
I have not checked that my changes have affected the application, they shouldent have, but just check that the markers are rendered.

### Specific Reviewer testing steps (Optional)
Run tests - check they all pass
check update script works

1. Set a small number of average prices to 0, e.g. BN23 7
2. Run the update script using that prefix
3. Check the values are set, and no errors occur


---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/gregam3/asegroup1/240)
